### PR TITLE
isogram: Revise canonical test data

### PR DESCRIPTION
**Pull request by behrtam**
This removes the two unicode test cases as discussed in #428. One test case was an isogram with German umlauts, the other one was a French word with a duplicated character with accent (also non-ASCII). I don't think we loose anything here without those test cases that isn't covered by the remaining ones.

I also added descriptions for the test cases and a test case with an empty string. *If you would prefer those non unicode changes in a separate PR, just ask.*

### CODE REVIEW

**Comment by Insti** [comment]
Awesome, looks good. thanks @behrtam

File: exercises/isogram/canonical-data.json [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
-  }]
+  "cases": [
+    {
+      "description": "empty string has no duplicates",
+      "input": "",
+      "expected": true
+    },
+    {
+      "description": "isogram with only lower case characters",
+      "input": "duplicates",
```
Comment (Insti) [inline exercises/isogram/canonical-data.json]: I'd prefer to use a different word here as 'duplicates' is misleading. (since there are no duplicates.)
`isogram` could be good, since `isogram` is an isogram

**Comment by Insti** [comment]
Including a test case with spaces might also be good.

"the quick brown fox jumps over the lazy dog"
Might be interesting since it is a panagram, but not an isogram.

"no duplicates"
Is an isogram and is autological

**Comment by Insti** [comment]
(sorry to hijack your PR with potentially unrelated changes.)

File: exercises/isogram/canonical-data.json (hunk 1 above)
Comment (behrtam) [inline exercises/isogram/canonical-data.json]: It its kind of funny a bit like that isogram**s** is no isogram (in the readme). But I agree that we should change that.

**Comment by behrtam** [comment]
The last test case `"Emily Jung Schwartzkopf"` has two white spaces, so you thing we should add a negative test case with white spaces?

**Comment by kytrinyx** [comment]
@behrtam I think the `"Emily Jung Schwartzkopf"` case covers it.

**Comment by Insti** [comment]
Thanks @behrtam :heart:
